### ISSUE TITLE
Maybe an error in fft.c

### ISSUE THREAD

**Issue by SebastienDorgan** [jtfell/c-fft#1]
I think line 41 of fft.c is wrong: (for(z = 0; z < 30; z++) ) should be for(z = 0; z < N; z++) , doesn't it?

**Comment by jtfell** [comment]
Yep I think you're right @SebastienDorgan. I haven't used this lib in many years and only wrote it for a university assignment. Feel free to leave a PR to fix it if you want but I really don't recommend using this for production uses
